cli/flags.go

```diff
@@ -53,6 +53,11 @@ var (
 		Value: "25s",
 		Usage: "period between each heartbeat",
 	}
+	flTTL = cli.StringFlag{
+		Name:  "time-to-live, ttl",
```

Comment:
Also, how about rounding out?

What do you think of those defaults:
- `heartbeat` = 20s
- `ttl` = 60s